README.md

```diff
@@ -0,0 +1,136 @@
+
```

Comment:
Unlike more of our other repos, the README here is also sent out & published on the PyPI repo (the python package distribution system). Consequently, it needed be as long or complicated.  Most of the details are relegated to `api.md` in docs.